### PR TITLE
UP5 BugFixes

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
- https://issues.cask.co/browse/CDAP-11952
- https://issues.cask.co/browse/CDAP-11992
- https://issues.cask.co/browse/CDAP-11999
- https://issues.cask.co/browse/CDAP-12031

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/ColumnsTab/ColumnsTabDetail.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default function ColumnsTabDetail({columnInfo}) {
   let headers = Object.keys(types);
   return (
     <tr className="column-tab-details">
+      <td></td>
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/ColumnsTab/ColumnsTabDetail.js]: Nit: uses self-closing tags for the newly added `td` and `th` tags?

File: cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/DatabaseBrowser/index.js [hunk 2]
```diff
@@ -116,6 +123,44 @@ export default class DatabaseBrowser extends Component {
       );
   }
 
+  fetchTables() {
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/DatabaseBrowser/index.js]: I see this new function being created, but I don't see it being used anywhere?

**Comment by ajainarayanan** [comment]
@elfenheart's changes LGTM 👍

**Comment by tonybach** [comment]
Code LGTM 👍

**Comment by bdmogal** [comment]
@elfenheart unrelated, but can you change the ordering of the dropdown menu as follows:
1. Move Find and Replace and Fill Null or Empty to the same group as Format and Filter.
2. Move Mask Data to the same group as Encode and Decode.

**Comment by bdmogal** [comment]
@elfenheart after following the steps for CDAP-11992, I get this:
image

**Comment by elfenheart** [comment]
@bdmogal is it stuck or is it actually loading? those steps for CDAP-11992 takes a long time to load

**Comment by bdmogal** [comment]
Its stuck. You can see the error in the background, which seems to suggest that the request completed.

**Comment by bdmogal** [comment]
@elfenheart it seems like for CDAP-11999, the particular issue was fixed, but I noticed that the number of tabs being displayed does not change when the side panel is closed. See attached gif.
num-tabs

**Comment by bdmogal** [comment]
LGTM

**Pull request merged**